In [1]:
price = 4.50
count = 3
total = price * count
print(f"{count} coffees cost {total} dollars")

3 coffees cost 13.5 dollars


In [2]:
scores = [72, 85, 91, 64, 88]
high = 0
for s in scores:
    if s >= 80:
        high = high + 1
print(high)

3


In [3]:
def to_inches(cm):
    return cm / 2.54

print(round(to_inches(100), 1))

39.4


In [9]:
name = "John"
hourly_wage = 15.65
hours = 35
weekly_pay = hourly_wage * hours
print(f"{name} worked {hours} hours and earned ${weekly_pay} this week.")


John worked 35 hours and earned $547.75 this week.


In [12]:
temps = [61, 65, 72, 58, 70, 74, 68]
high = 0
for t in temps:
    if t > 65:
        high = high + 1
print(high)

4


In [15]:
def F_to_C(f):
    c = (f-32)*5/9
    return round(c,1)
F_to_C(temps[0])

16.1

In [16]:
movie = {"title": "Moneyball", "year": 2011, "rating": 7.6}
for key, value in movie.items():
    print(key, ":", value)

title : Moneyball
year : 2011
rating : 7.6


In [1]:
import pandas as pd
df = pd.read_csv("chickwts.csv")
df.head()

,Unnamed: 0,weight,feed
0,1,179,horsebean
1,2,160,horsebean
2,3,136,horsebean
3,4,227,horsebean
4,5,217,horsebean


In [3]:
df.shape

(71, 3)

In [5]:
df.describe()

,Unnamed: 0,weight
count,71.000000,71.000000
mean,36.000000,261.309859
std,20.639767,78.073700
min,1.000000,108.000000
25%,18.500000,204.500000
50%,36.000000,258.000000
75%,53.500000,323.500000
max,71.000000,423.000000


The most interesting thing about this dataset to me is the spread of the weights. The largest chick in the dataset is nearly four times as heavy as the lightest. This is also reflected by the relatively large standard deviation of the dataset (78.1) means the coefficient of variance (stdev/mean) is roughly 30%.